javascripts/discourse/components/home-list.gjs

```diff
@@ -52,6 +54,69 @@ export default class HomeList extends Component {
     );
   }
 
+  get promoConfig() {
+    return settings.promo_tile[0] || {};
+  }
```

Comment:
```suggestion
  @cached
  get promoConfig() {
    return settings.promo_tile[0] || {};
  }
```

(requires `import { cached } from "@glimmer/tracking"`)

This is a micro-optimization but every time you reference `this.promoConfig` you create a new empty object